### PR TITLE
[zh-cn] sync blog native-sidecar

### PR DESCRIPTION
**Pull request by vsxen**
2023-08-25-sidecar-kep.md

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+layout: blog
+title: "Kubernetes 1.28： 介绍原生 sidecar 容器"
+date: 2023-08-25
+slug: native-sidecar-containers
+---
+
+<!-- layout: blog
+title: "Kubernetes v1.28: Introducing native sidecar containers"
+date: 2023-08-25
+slug: native-sidecar-containers -->
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:11]:
```suggestion
slug: native-sidecar-containers
-->
```

**Review by windsonsea — COMMENTED** [review]
Please refer to [zh-cn Localization style](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+***作者*** Todd Neal (AWS), Matthias Bertschy (ARMO), Sergey Kanzhelev (Google), Gunju Kim (NAVER), Shannon Kularathna (Google)
+
+<!-- ***Authors:*** Todd Neal (AWS), Matthias Bertschy (ARMO), Sergey Kanzhelev (Google), Gunju Kim (NAVER), Shannon Kularathna (Google) -->
+
+
+本文介绍了如何使用新的Sidecar功能，该功能允许可重新启动的init容器，并已在Kubernetes 1.28中以alpha版本提供。我们希望听取你的反馈，以便尽快将此功能推进到毕业阶段。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:18]:
```suggestion
本文介绍如何使用新的边车特性；该特性允许设置可重新启动的 Init 容器，并已在 Kubernetes 1.28
中以 Alpha 状态提供。我们希望听取你的反馈，以便尽快将此功能推进到毕业阶段。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+<!-- This post explains how to use the new sidecar feature, which enables restartable init containers and is available in alpha in Kubernetes 1.28. We want your feedback so that we can graduate this feature as soon as possible. -->
+
+“sidecar” 的概念几乎从一开始就是 Kubernetes 的一部分。 2015 年，一篇关于 sidecars 容器的 [博客文章](/blog/2015/06/the-distributed-system-toolkit-patterns/) 将 sidecar 描述为“扩展和增强‘主’容器”的附加容器。 Sidecar 容器已成为一种常见的 Kubernetes 部署模式，通常用于网络代理或作为日志系统的一部分。到目前为止，sidecar 是 Kubernetes 用户在没有原生支持的情况下应用的概念。缺乏本机支持导致了一些使用摩擦，此增强功能旨在解决这些问题。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:22]:
```suggestion
“边车（Sidecar)” 的概念几乎从一开始就是 Kubernetes 的一部分。2015 年，
一篇关于边车容器的[博客文章](/blog/2015/06/the-distributed-system-toolkit-patterns/)将边车描述为
“扩展和增强‘主’容器”的附加容器。边车容器已成为一种常见的 Kubernetes 部署模式，
通常用于网络代理或作为日志系统的一部分。到目前为止，边车一直是 Kubernetes
用户在缺少原生支持的情况下应用的概念。缺乏原生支持也导致了一些使用摩擦，
此增强特性旨在解决这些问题。
```

**Review by tengqm — COMMENTED** [review]
Please follow our style for ease of review.

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md (hunk 2 above)
Comment (vsxen) [reply content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:18]: 这个Sidecar 属于专业术语吧？

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+在 alpha 阶段，我们希望你在你的环境中尝试 sidecar 容器，并在遇到错误或异常点时提出问题。我们对以下方面的反馈特别感兴趣：
+
+<!-- 
+- The shutdown sequence, especially with multiple sidecars running 
+- The backoff timeout adjustment for crashing sidecars 
+- The behavior of Pod readiness and liveness probes when sidecars are running 
+-->
+- 关闭顺序，尤其是多个 sidecar 一起运行的时候
+- sidecar 的重启回退超时时间
+- Sidecar 运行时 Pod 就绪性和活性探测的行为
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:192]:
```suggestion
- 关闭顺序，尤其是多个边车一起运行的时候
- 边车的重启回退超时时间调整
- 边车运行时 Pod 就绪性和存活性探针的行为
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+
+
+<!-- 
+To open an issue, see the [Kubernetes GitHub repository](https://github.com/kubernetes/kubernetes/issues/new/choose). 
+-->
+要提出问题，请参阅 [Kubernetes GitHub 存储库](https://github.com/kubernetes/kubernetes/issues/new/choose)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:198]:
```suggestion
要登记问题，请访问 [Kubernetes GitHub 仓库](https://github.com/kubernetes/kubernetes/issues/new/choose)。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+
+<!-- 
+## What’s next? 
+-->
+## 下一步是什么？
+
+<!-- 
+In addition to the known issues that will be resolved, we're working on adding termination ordering for sidecar and main containers. This will ensure that sidecar containers only terminate after the Pod's main containers have exited. 
+-->
+除了将要解决的已知问题之外，我们正在努力为 sidecar 和主容器添加终止顺序。这将确保 sidecar 容器仅在 Pod 的主容器退出后终止。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:208]:
```suggestion
除了要解决的已知问题之外，我们正在努力为边车和主容器添加终止顺序。
终止顺序能够确保边车容器仅在 Pod 的主容器退出后才终止。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+
+
+<!-- 
+We’re excited to see the sidecar feature come to Kubernetes and are interested in feedback. 
+-->
+我们很高兴看到 Kubernetes 引入了 sidecar 功能，并对反馈感兴趣。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:214]:
```suggestion
我们很高兴看到 Kubernetes 引入了边车特性，并对反馈感兴趣。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+
+<!-- 
+## Acknowledgements 
+-->
+## 致谢
+
+<!-- 
+Many years have passed since the original KEP was written, so we apologize if we omit anyone who worked on this feature over the years. This is a best-effort attempt to recognize the people involved in this effort. 
+-->
+自最初的 KEP 编写以来已经过去了很多年，但是我们忽略了多年来致力于此功能的相关人员，我们对此深表歉意。这是对参与这项工作的人们的尽最大努力的尝试。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:225]:
```suggestion
自最初的 KEP 编写以来已经过去了很多年，因此如果我们漏掉了多年来致力于此特性的相关人员，请接受我们的道歉。
我们尽最大努力去认可参与这项工作的人们。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+- [jpbetz](https://github.com/Jpbetz) for API and termination ordering designs as well as code reviews
+- [Joseph-Irving](https://github.com/Joseph-Irving) and [rata](https://github.com/rata) for the early iterations design and reviews years back
+- [swatisehgal](https://github.com/swatisehgal) and [ffromani](https://github.com/ffromani) for early feedback on resource managers impact
+- [alculquicondor](https://github.com/Alculquicondor) for feedback on addressing the version skew of the scheduler
+- [wojtek-t](https://github.com/Wojtek-t) for PRR review of a KEP
+- [ahg-g](https://github.com/ahg-g) for reviewing the scheduler portion of a KEP
+- [adisky](https://github.com/Adisky) for the Job completion issue 
+-->
+
+- [mrunalp](https://github.com/mrunalp/) 用于设计讨论和评论
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:244]:
```suggestion
- [mrunalp](https://github.com/mrunalp/) 参与了设计讨论和评审 
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+- [swatisehgal](https://github.com/swatisehgal) and [ffromani](https://github.com/ffromani) for early feedback on resource managers impact
+- [alculquicondor](https://github.com/Alculquicondor) for feedback on addressing the version skew of the scheduler
+- [wojtek-t](https://github.com/Wojtek-t) for PRR review of a KEP
+- [ahg-g](https://github.com/ahg-g) for reviewing the scheduler portion of a KEP
+- [adisky](https://github.com/Adisky) for the Job completion issue 
+-->
+
+- [mrunalp](https://github.com/mrunalp/) 用于设计讨论和评论 
+- [thockin](https://github.com/thockin/) 多年来的 API 讨论和支持 
+- [bobbypage](https://github.com/bobbypage) 审查
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:246]:
```suggestion
- [bobbypage](https://github.com/bobbypage) 的评审
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+- [smarterclayton](https://github.com/smarterclayton) 进行详细审查和反馈
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:247]:
```suggestion
- [smarterclayton](https://github.com/smarterclayton) 的细致评审和反馈 
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+- [wojtek-t](https://github.com/Wojtek-t) for PRR review of a KEP
+- [ahg-g](https://github.com/ahg-g) for reviewing the scheduler portion of a KEP
+- [adisky](https://github.com/Adisky) for the Job completion issue 
+-->
+
+- [mrunalp](https://github.com/mrunalp/) 用于设计讨论和评论 
+- [thockin](https://github.com/thockin/) 多年来的 API 讨论和支持 
+- [bobbypage](https://github.com/bobbypage) 审查
+- [smarterclayton](https://github.com/smarterclayton) 进行详细审查和反馈 
+- [howardjohn](https://github.com/howardjohn) 多年来的反馈并在实施过程中尽早尝试
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:248]:
```suggestion
- [howardjohn](https://github.com/howardjohn) 多年来的反馈以及在实现过程中的早期尝试
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 13]
```diff
@@ -0,0 +1,270 @@
...
+- [derekwaynecarr](https://github.com/derekwaynecarr) 和 [dchen1107](https://github.com/dchen1107) 领导力
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:249]:
```suggestion
- [derekwaynecarr](https://github.com/derekwaynecarr) 和 [dchen1107](https://github.com/dchen1107) 的领导
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 14]
```diff
@@ -0,0 +1,270 @@
...
+- [adisky](https://github.com/Adisky) for the Job completion issue 
+-->
+
+- [mrunalp](https://github.com/mrunalp/) 用于设计讨论和评论 
+- [thockin](https://github.com/thockin/) 多年来的 API 讨论和支持 
+- [bobbypage](https://github.com/bobbypage) 审查
+- [smarterclayton](https://github.com/smarterclayton) 进行详细审查和反馈 
+- [howardjohn](https://github.com/howardjohn) 多年来的反馈并在实施过程中尽早尝试
+- [derekwaynecarr](https://github.com/derekwaynecarr) 和 [dchen1107](https://github.com/dchen1107) 领导力 
+- [jpbetz](https://github.com/Jpbetz) 用于 API 和终止排序设计以及代码审查
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:250]:
```suggestion
- [jpbetz](https://github.com/Jpbetz) 对 API 和终止排序的设计以及代码评审
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 15]
```diff
@@ -0,0 +1,270 @@
...
+- [Joseph-Irving](https://github.com/Joseph-Irving) 和 [rata](https://github.com/rata) 用于多年前的早期迭代设计和审查
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:251]:
```suggestion
- [Joseph-Irving](https://github.com/Joseph-Irving) 和 [rata](https://github.com/rata) 参与多年前的早期迭代设计和评审
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 16]
```diff
@@ -0,0 +1,270 @@
...
+
+- [mrunalp](https://github.com/mrunalp/) 用于设计讨论和评论 
+- [thockin](https://github.com/thockin/) 多年来的 API 讨论和支持 
+- [bobbypage](https://github.com/bobbypage) 审查
+- [smarterclayton](https://github.com/smarterclayton) 进行详细审查和反馈 
+- [howardjohn](https://github.com/howardjohn) 多年来的反馈并在实施过程中尽早尝试
+- [derekwaynecarr](https://github.com/derekwaynecarr) 和 [dchen1107](https://github.com/dchen1107) 领导力 
+- [jpbetz](https://github.com/Jpbetz) 用于 API 和终止排序设计以及代码审查
+- [Joseph-Irving](https://github.com/Joseph-Irving) 和 [rata](https://github.com/rata) 用于多年前的早期迭代设计和审查 
+- [swatisehgal](https://github.com/swatisehgal) 和 [ffromani](https://github.com/ffromani) 获取有关资源管理器影响的早期反馈
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:252]:
```suggestion
- [swatisehgal](https://github.com/swatisehgal) 和 [ffromani](https://github.com/ffromani) 针对有关资源管理器影响所给出的早期反馈 
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 17]
```diff
@@ -0,0 +1,270 @@
...
+- [mrunalp](https://github.com/mrunalp/) 用于设计讨论和评论 
+- [thockin](https://github.com/thockin/) 多年来的 API 讨论和支持 
+- [bobbypage](https://github.com/bobbypage) 审查
+- [smarterclayton](https://github.com/smarterclayton) 进行详细审查和反馈 
+- [howardjohn](https://github.com/howardjohn) 多年来的反馈并在实施过程中尽早尝试
+- [derekwaynecarr](https://github.com/derekwaynecarr) 和 [dchen1107](https://github.com/dchen1107) 领导力 
+- [jpbetz](https://github.com/Jpbetz) 用于 API 和终止排序设计以及代码审查
+- [Joseph-Irving](https://github.com/Joseph-Irving) 和 [rata](https://github.com/rata) 用于多年前的早期迭代设计和审查 
+- [swatisehgal](https://github.com/swatisehgal) 和 [ffromani](https://github.com/ffromani) 获取有关资源管理器影响的早期反馈 
+- [alculquicondor](https://github.com/Alculquicondor) 获取有关解决调度程序版本偏差的反馈 - [wojtek-t](https://github.com/Wojtek-t) 用于 KEP 的 PRR 审查
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:253]:
```suggestion
- [alculquicondor](https://github.com/Alculquicondor) 针对与调度程序版本偏差相关的反馈 - [wojtek-t](https://github.com/Wojtek-t) 对 KEP 的 PRR 的评审
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 18]
```diff
@@ -0,0 +1,270 @@
...
+- [thockin](https://github.com/thockin/) 多年来的 API 讨论和支持 
+- [bobbypage](https://github.com/bobbypage) 审查
+- [smarterclayton](https://github.com/smarterclayton) 进行详细审查和反馈 
+- [howardjohn](https://github.com/howardjohn) 多年来的反馈并在实施过程中尽早尝试
+- [derekwaynecarr](https://github.com/derekwaynecarr) 和 [dchen1107](https://github.com/dchen1107) 领导力 
+- [jpbetz](https://github.com/Jpbetz) 用于 API 和终止排序设计以及代码审查
+- [Joseph-Irving](https://github.com/Joseph-Irving) 和 [rata](https://github.com/rata) 用于多年前的早期迭代设计和审查 
+- [swatisehgal](https://github.com/swatisehgal) 和 [ffromani](https://github.com/ffromani) 获取有关资源管理器影响的早期反馈 
+- [alculquicondor](https://github.com/Alculquicondor) 获取有关解决调度程序版本偏差的反馈 - [wojtek-t](https://github.com/Wojtek-t) 用于 KEP 的 PRR 审查 
+- [ahg-g](https://github.com/ahg-g) 用于审查 KEP 的调度程序部分
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:254]:
```suggestion
- [ahg-g](https://github.com/ahg-g) 对 KEP 的调度器部分的评审
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 19]
```diff
@@ -57,82 +57,84 @@ spec:
...
 - Init containers have a well-defined startup order regardless of whether you set a `restartPolicy`, so you can ensure that your sidecar starts before any container declarations that come after the sidecar declaration in your manifest.
 - Sidecar containers don't extend the lifetime of the Pod, so you can use them in short-lived Pods with no changes to the Pod lifecycle.
 - Sidecar containers are restarted on exit, which improves resilience and lets you use sidecars to provide services that your main containers can more reliably consume. 
 -->
-- 不管你是否设置了 `restartPolicy`，Init 容器都有明确定义的启动顺序，因此你可以确保在清单中的 Sidecar 声明之后，其他容器声明之前启动 Sidecar 容器。
-- Sidecar 容器不会延长 Pod 的生命周期，因此你可以在短期 Pod 中使用它们，而无需更改 Pod 生命周期。
-- Sidecar 容器在退出时重新启动，这提高了弹性，并允许你使用 sidecar 来提供主容器可以更可靠地使用的服务。
+- 不管你是否设置了 `restartPolicy`，Init 容器都有明确定义的启动顺序，因此，你可以确保清单中的边车容器会在边车声明之后的所有容器之前启动。
+- 边车容器不会延长 Pod 的生命周期，因此你可以在生命期较短的 Pod 中使用它们，而无需更改 Pod 生命周期。
+-  边车容器在退出时会重新启动，这提高了可靠性，从而允许你使用边车来为主容器提供更为可靠的服务。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:89]:
```suggestion
- 边车容器在退出时会重新启动，这提高了可靠性，从而允许你使用边车来为主容器提供更为可靠的服务。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 20]
```diff
@@ -162,15 +165,15 @@ We recommend only using the sidecars feature gate in [short lived testing cluste
...
 
 <!-- 
 - The CPU, memory, device, and topology manager are unaware of the sidecar container lifetime and additional resource usage, and will operate as if the Pod had lower resource requests than it actually does.
 - The output of `kubectl describe node` is incorrect when sidecars are in use. The output shows resource usage that's lower than the actual usage because it doesn't use the new resource usage calculation for sidecar containers. 
 -->
-- CPU、内存、设备和拓扑管理器无法意识到 Sidecar 容器的生命周期和额外资源使用情况，它们会按照 Pod 较低的资源请求来运行，而不是实际使用的。
--- 使用 sidecar 时，`kubectl describe node` 命令描述节点的输出不正确。输出显示的资源使用量低于实际使用量，因为它没有对 sidecar 容器的资源使用量做计算。
+- CPU、内存、设备和拓扑管理器无法意识到边车容器的生命周期和额外资源使用情况，它们会按照 Pod 资源请求比实际用量低的假设来运行 Pod。
+-- 使用边车时，`kubectl describe node` 命令描述的节点的输出不正确。
+  输出显示的资源使用量低于实际使用量，因为它没有计算边车容器的资源使用量
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:176]:
```suggestion
  输出显示的资源使用量低于实际使用量，因为它没有计算边车容器的资源使用量。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 21]
```diff
@@ -145,13 +147,14 @@ The built-in sidecar feature solves for the use case of having a lifetime equal
...
 -->
-## 将现有 sidecar 过渡到新模型
+## 将现有边车过渡到新模型
 
 <!-- 
 We recommend only using the sidecars feature gate in [short lived testing clusters](/docs/reference/command-line-tools-reference/feature-gates/#feature-stages) at the alpha stage. If you have an existing sidecar that is configured as a main container so it can run for the lifetime of the pod, it can be moved to the `initContainers` section of the pod spec and given a `restartPolicy` of `Always`. In many cases, the sidecar should work as before with the added benefit of having a defined startup ordering and not prolonging the pod lifetime. 
 -->
 
-我们建议仅在 alpha 阶段的[短期测试集群](/docs/reference/command-line-tools-reference/feature-gates/#feature-stages) 中使用 sidecars 功能门。如果你有一个现有的 sidecar，被配置为主容器，以便它可以在 Pod 的生命周期内运行，则可以将其移至 Pod 规范的“initContainers”部分，并将“restartPolicy”指定为“Always”。在许多情况下，Sidecar 继续像以前一样工作，并且额外的好处是可以定义启动顺序，并且不会延长 Pod 的生命周期。
+我们建议在 Alpha 阶段仅对[短期存在的测试集群](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/#feature-stages)启用边车特性门控。如果你已经有一个被被配置为主容器的边车，且它可以在 Pod 的整个生命周期内运行，则可以将其移至 Pod 规约的 `initContainers` 部分，并将 `restartPolicy` 设置为 `Always`。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:156]: 建议对过长的行执行手动换行，便于变更追踪和评审。请彻查。

```suggestion
我们建议在 Alpha 阶段仅对[短期存在的测试集群](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/#feature-stages)启用边车特性门控。
如果你已经有一个被被配置为主容器的边车，且它可以在 Pod 的整个生命周期内运行，
则可以将其移至 Pod 规约的 `initContainers` 部分，并将 `restartPolicy` 设置为 `Always`。
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 22]
```diff
@@ -26,19 +26,19 @@ This post explains how to use the new sidecar feature, which enables restartable
 <!-- 
 The concept of a “sidecar” has been part of Kubernetes since nearly the very beginning. In 2015, sidecars were described in a [blog post](/blog/2015/06/the-distributed-system-toolkit-patterns/) about composite containers as additional containers that “extend and enhance the ‘main’ container”. Sidecar containers have become a common Kubernetes deployment pattern and are often used for network proxies or as part of a logging system. Until now, sidecars were a concept that Kubernetes users applied without native support. The lack of native support has caused some usage friction, which this enhancement aims to resolve.
  -->
-“边车（Sidecar)” 的概念几乎从一开始就是 Kubernetes 的一部分 2015 年，一篇关于 sidecars 容器的 [博客文章](/blog/2015/06/the-distributed-system-toolkit-patterns/) 将边车
+“边车（Sidecar)” 的概念几乎从一开始就是 Kubernetes 的一部分 2015 年，一篇关于边车容器的 [博客文章](/blog/2015/06/the-distributed-system-toolkit-patterns/) 将边车
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:29]:
```suggestion
“边车（Sidecar)” 的概念几乎从一开始就是 Kubernetes 的一部分。
2015 年，一篇关于边车容器的[博客文章](/blog/2015/06/the-distributed-system-toolkit-patterns/)将边车
```

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md [hunk 23]
```diff
@@ -26,19 +26,19 @@ This post explains how to use the new sidecar feature, which enables restartable
...
 ## What are sidecar containers in 1.28? 
 -->
-## 在 Kubernetes 1.28 中的 sidecar 容器是什么？
+## 在 Kubernetes 1.28 中的边车容器是什么？
 
 <!-- 
 Kubernetes 1.28 adds a new `restartPolicy` field to [init containers](/docs/concepts/workloads/pods/init-containers/) that is available when the `SidecarContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled. 
 -->
-Kubernetes 1.28 在 [init 容器](/docs/concepts/workloads/pods/init-containers/) 中添加了一个新的 `restartPolicy` 字段，该字段可以在 `SidecarContainers` [特性门控](/docs/reference/command-line-tools-reference/feature-gates/) 被打开时使用。
+Kubernetes 1.28 在 [Init 容器](/zh-cn/docs/concepts/workloads/pods/init-containers/) 中添加了一个新的 `restartPolicy` 字段，该字段可以在 `SidecarContainers` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/) 启用时使用。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:41]:
```suggestion
Kubernetes 1.28 在 [Init 容器](/zh-cn/docs/concepts/workloads/pods/init-containers/)
中添加了一个新的 `restartPolicy` 字段，该字段可以在 `SidecarContainers`
[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)启用时使用。
```

**Comment by tengqm** [comment]
Please squash your PR into a single commit.

File: content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md (hunk 21 above)
Comment (vsxen) [reply content/zh-cn/blog/_posts/2023-08-25-sidecar-kep.md:156]: 好的

**Comment by windsonsea** [comment]
If you have modified by following the suggested comments, you can click the `Resolve conversation` button, so every reviewer can easily know the progress once he/she jumps in.

**Comment by windsonsea** [comment]
/approve

Some nits can be corrected later.

**Pull request merged**